ninchatsdk/src/main/java/com/ninchat/sdk/helper/siteconfigparser/NinchatSiteConfig.kt

```diff
@@ -306,13 +306,13 @@ class NinchatSiteConfig {
         getTranslation("Close")
 
     fun getConferenceTitleText(): String =
-        getTranslation("Video meeting")
+        getTranslation("This is a video meeting channel")
 
     fun getConferenceButtonText(): String =
         getTranslation("Join video meeting")
 
     fun getConferenceDescriptionText(): String =
-        getTranslation("Continue to join video meeting. You can toggle text chat from the top.")
+        getTranslation("translations", "meeting description") ?: ""
```

Comment:
This should come from site config and not from translations. It should come from key "videoMeetingInfoText" and can contain html. If said key is not available, then it is not displayed